src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -448,7 +568,30 @@ def option(self, key: str, value: Any) -> "DataFrameReader":
         Args:
             key: Name of the option (e.g. ``compression``, ``skip_header``, etc.).
             value: Value of the option.
+        Note: Aliases are supported for specifying options in the `option` method.
+
+        The following option aliases are available:
+
+        - HEADER: Accepted values are True or False which will be interpreted as `option('SKIP_HEADER',1)` or `option('SKIP_HEADER',0)`.
+        - DELIMITER: will be interpreted as `FIELD_DELIMITER`.
+        - SEP: will be interpreted also as `FIELD_DELIMITER`.
+        - LINESEP: will be interpreted as `RECORD_DELIMITER`.
+        - PATHGLOBFILTER: will be interpreted as `PATTERN`.
+        - QUOTE: will be interpreted as `FIELD_OPTIONALLY_ENCLOSED_BY`.
+        - NULLVALUE: will be interpreted as `NULL_IF`.
+        - DATEFORMAT: will be interpreted as `DATE_FORMAT`.
+        - TIMESTAMPFORMAT: will be interpreted as `TIMESTAMP_FORMAT`.
+        - INFERSCHEMA: will be interpreted as `INFER_SCHEMA`.
+
         """
+        if key.upper() in ["FORMAT", "TYPE"]:
+            return self.format(value)
+        elif key.upper() == "SCHEMA":
+            return self.schema(value)
```

Comment:
does pyspark support `schema` and `format` via `options`?

Reply:
I have that idea but will need to check. Checking the spark code format seems to be stored inside a different field and options in extraOptions but will double check

Reply:
My tests indicated that it is not supported as part of the options.